lib/jsonnet/evaluator.rb

```diff
@@ -0,0 +1,46 @@
+module Jsonnet
+  class Evaluator
```

Comment:
The purpose of this class is mostly to take options in a form of hash (which is what the top level API can be provided with) and know how to apply them for the VM, whether they be instance methods or method arguments.

It's meant as a private class that the top level API uses.

I can look into adding these to the VM, see if they don't add too much bulk.